.azure-pipelines/templates/copy-and-build.yml

```diff
@@ -3,17 +3,17 @@ steps:
   inputs:
     SourceFolder: '$(Agent.TempDirectory)'
     Contents: '**'
```

Comment:
this input is not used, remove

Reply:
Sorry, which input?

Reply:
target folder

Reply:
nevermind, this comment, I just realized I was ready this wrong.